channel-policies.md

```diff
@@ -15,7 +15,7 @@ Here you can find postings for development jobs in the Triangle area
 
 Rules:
 
-1. Do not send unsolicited DMs to members about job openings
+1. Do not send unsolicited DMs or emails to members about job openings
```

Comment:
What about using the term "communications" in order to make this a generic enough policy to cover other types of communication methods?

`Do not send unsolicited communications (DMs, emails, calls, etc) to members about job openings or other commercial solicitations`